proc/syscall_windows.go

```diff
@@ -22,8 +22,53 @@ type _THREAD_BASIC_INFORMATION struct {
 	BasePriority   int32
 }
 
+type _CREATE_PROCESS_DEBUG_INFO struct {
+	File                syscall.Handle
+	Process             syscall.Handle
+	Thread              syscall.Handle
+	BaseOfImage         uintptr
+	DebugInfoFileOffset uint16
+	DebugInfoSize       uint16
+	ThreadLocalBase     uintptr
+	StartAddress        uintptr
+	ImageName           *byte
```

Comment:
This part of the doc you quoted is what I was referring to:

> in the address space of the process being debugged

So to read from this pointer requires a ReadProcessMemory call against the debuggee process.

That said - as described in http://www.codeproject.com/Articles/43682/Writing-a-basic-Windows-debugger this field is mostly not useful anyway. :-)